Dockerfile.windows

```diff
@@ -57,12 +57,18 @@ RUN Write-Host ('Installing Visual C++ Redistributable Package'); `
 
 #
 # Install OpenSSL
+# This must be done after installing Visual Studio
 #
 ADD https://slproweb.com/download/Win64OpenSSL-1_1_1L.msi /local/Win64OpenSSL-1_1_1L.msi
 
-RUN Start-Process msiexec.exe `
-    -ArgumentList '/i C:\local\Win64OpenSSL-1_1_1L.msi ', '/quiet ', `
-    '/norestart ' -NoNewWindow -Wait;
+# Hash copied from https://github.com/slproweb/opensslhashes
+RUN $ExpectedHash = \"1e62e6459835f61c418532a1544e295be5400498b3569aadfb6d0b5d7a1c1dbc\";`
```

Comment:
You may be able to use single quotes to avoid the backslashes…

Reply:
AH YOU'RE RIGHT! and since i changed "doesn't match" to "mismatch", i don't have to escape anything at alllll 🤸‍♀️

Reply:
wait jk, it doesn't expand the vars with single quotes